Automations/app-workspace/README.md

```diff
@@ -0,0 +1,24 @@
+# App workspace
+
+Say you have the boring task of manually opening apps for your specific purpose (work, entertainment, development,...). Sometimes you may even forget how to launch a program (if it is not added to your path or is tricky to open) and that leads to more time-consuming to prepare your apps-workspace.  
+This program will help you save valuable time by saving all your running apps to a file and then later allow you to load all these apps at once with a single command. It even allows you to launch those apps in a new virtual desktop (if your computer supports it).
+
+### Additional packages 📝
+
+- pyvda
+- pyautogui
+
+#### But how do I install additonal packages? 🤨
```

Comment:
typo in readme
```suggestion
#### But how do I install additional packages? 🤨
```

Reply:
I just found out that this typo exist in template readme also, thanks for bringing this to light